test/test.cpp

```diff
@@ -508,6 +508,17 @@ BOOST_AUTO_TEST_CASE(time_fromString)
 
 }
 
+BOOST_AUTO_TEST_CASE(time_toTimeValues)
+{
+    base::Time time = base::Time::fromMicroseconds(90061001001);
```

Comment:
Great that you added tests ....

But I believe it would be best to have different values for each field. This test for instance would not catch having reported the milliseconds as days or seconds as microseconds or ...

Reply:
Good point. Will change that.